custom_components/tuya_local/devices/dynasty_btx_fireplace.yaml

```diff
@@ -0,0 +1,193 @@
+name: Fireplace
+products:
+  - id: 5081205010521cfac95f
+    name: Dynasty BTX Fireplace
+primary_entity:
+  entity: climate
+  icon: "mdi:fireplace"
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat
+    - id: 3
+      name: current_temperature
+      type: integer
+      range:
```

Comment:
No range needed on read-only attributes, what is received from the device will be displayed, regardless of whether it is in range or not.